contents/handbook/people/team-structure/core-experience.md

```diff
@@ -24,51 +27,48 @@ organization in matters of product design, user experience, and frontend & API t
 
 [See team structure page](/handbook/people/team-structure/team-structure)
 
+
 ## Mission
 
+Company goal that resonates most with this team:
+
+> Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch.
+
 Team Core Experience aims to:
 
-- Implement PostHog's vision of increasing the number of successful products in the world, by building features that will move us closer to said vision.
+- Increase the quality of our features across all dimensions (see scope below).
 - Maintain a coherent strategy for frontend and API development (technologies, tools and methodology).
-- Always be in touch with product leadership and our customers, in order to figure out what works, what doesn't, and what needs to be built next.
-- Actually build those features.
-- Support other teams in PostHog in maintaining a coherently high level of product experience across the board.
-- Push the envelope and give back to the open source community whenever we develop abstractions that could benefit other organizations.
+- Ensuring our features are used the way we expect them to.
+- Understanding users and their needs... and solving for that.
 
 ## Goals
 
-With the work above, we aim to solve the following meta-goals:
-
-- Part of our vision: It should be unbelievably easy and intuitive for anyone to use PostHog to get answers to  questions they have, and even find the answers to questions they had not considered asking.
+What is quality and how do we track it?
 
-- To make our tech worthy of the fanatical following that is emerging around PostHog.
-- Build the next generation of engineers and engineering leaders, who would be running things when we are at 100 developers.
-- Remain a highly flexible team, ready to adapt to changing business requirements.
-- Be part of formalizing "the PostHog way", and writing a book about it.
-- Our tech stack is rather unique, though pretty powerful. We are also in a really powerful position to drive change in the industry ([data-first frontend](https://kea.js.org/blog/data-first-frontend-revolution), anyone?). Let's capitalise on that.
-- Zero TypeScript errors, code covered by logic tests, all scenes in Storybook, etc.
-- To do a damn good job at all of the above.
+We track specific goals/targets depending on feature. For example:
 
-In short, if you want a crash course at how to build large and complex webapps, at scale, while retaining a maintainable codebase and high customer satisfaction, and touch every part of the stack, this is the team to work with.
+- Qualitative data (what users are saying)
+- Measured data (KPIs, performance metrics, etc)
+- Internal company feedback (meeting company goals)
 
+Main metric: number of discoveries.
```

Comment:
```suggestion
Main metric: number of [Discoveries](/handbook/product/metrics#what-is-a-discovery).
```

Reply:
Do we list other metrics?

@clarkus suggested user feedback. So could have a metric of number of user interviews? As a proxy for "user feedback gathered"

@rcmarron suggested "number of bugs" (IIRC). Could be gamed but there's something there. Something between number of sentry errors, number of issues opened, number of issues closed, speed of first response to issue, speed to close issue. Trying to think of two(ish) measures that have tension between them. So you can't just make it so users can't raise issues and then celebrate zero issues raised :)

Also quantitative measures -> page speed budget?

Reply:
Would love if we could incorporate another set of metrics, particularly to measure UX/perceived quality, however I'm not sure how we can measure this effectively in practice. Having a metric that's difficult to measure an influence seems worse than having no metric (e.g. # of interviews seems quite hard to improve by sheer effort of the Engineering Team)